overlord/hookstate/hooks.go

```diff
@@ -70,6 +71,22 @@ func SetupPreRefreshHook(st *state.State, snapName string) *state.Task {
 	return task
 }
 
+func SetupAutoRefreshGatingHook(st *state.State, snapName string, base, restart bool) *state.Task {
```

Comment:
shouldn't this name follow the hook name like the rest?  SetupGateAutoRefreshHook ?